R/class-workbook-wrappers.R

```diff
@@ -3087,64 +3164,64 @@ wb_add_form_control <- function(
 
 }
 
-#' Add conditional formatting to cells
+#' Add conditional formatting to cells in a worksheet
 #'
-#' Add conditional formatting to cells
-#' @param wb A workbook object
+#' Add conditional formatting to cells.
+#' You can find more details in `vignette("conditional-formatting")`.
+#'
+#' @param wb A Workbook object
 #' @param sheet A name or index of a worksheet
 #' @param dims A cell or cell range like "A1" or "A1:B2"
-#' @param rule The condition under which to apply the formatting. See examples.
-#' @param style A style to apply to those cells that satisfy the rule. Default is 'font_color = "FF9C0006"' and 'bgFill = "FFFFC7CE"'
-#' @param type The type of conditional formatting rule to apply.
-#' @param params Additional parameters passed.  See **Details** for more
+#' @param rule The condition under which to apply the formatting. See **Examples**.
+#' @param style A style to apply to those cells that satisfy the rule.
+#'   Default is `font_color = "FF9C0006"` and `bg_fill = "FFFFC7CE"`
```

Comment:
maybe add a word regarding ARGB (openxml uses the alpha channel first then RGB, whereas afaik the usual default is RGBA)

Reply:
Copied in details. But this can be reviewed later.